libs/designsystem/src/lib/components/icon/icon.component.ts

```diff
@@ -54,13 +57,33 @@ export class IconComponent implements OnChanges {
     }
   }
 
-  constructor(@Optional() @Inject(ICON_SETTINGS) private iconSettings?: IconSettings) {}
+  /**
+   * iconRegistryService: Registry for custom icons.
+   * iconSettings: @deprecated Use KirbyIconRegistryService for adding custom icons.
+   */
+  constructor(
+    private iconRegistryService: KirbyIconRegistryService,
+    @Optional() @Inject(ICON_SETTINGS) private iconSettings?: IconSettings
+  ) {
+    this.combineIconSettings();
+  }
+
+  private combineIconSettings() {
+    const iconSettings: Icon[] =
```

Comment:
This is not iconSettings but icons from icon settings

Reply:
👍